Oleksadr_Petrov/HW4/Task1.py

```diff
@@ -0,0 +1,19 @@
+def isEven(number):
+    if number % 2 == 0:
+        return True
+    else:
+        return False
+
+
+values = [1, 2, 3, 4, 5, 6, 7, 8]
+odd = []
+even = []
+
+for element in list:
+    if isEven(element):
+        even.append((list.index(element), element))
```

Comment:
what will happen if we have a list with duplicated numbers? 
For example: [1, 2, 3, 4, 5, 6, 7, 8, 1, 2]